bot/src/commands/add-inbox-item.js

```diff
@@ -0,0 +1,59 @@
+import { get, post } from '../utils';
+
+const CONFIRMATION_MESSAGES = [
+  'naturally',
+  'of course',
+  'splendid',
+  'jolly good',
+  'noted',
+  'point taken',
+  'at your service',
+  'sounds good'
+];
+
+const ERROR_MESSAGES = [
+  'uh-oh, that didn\'t work out',
+  'something went wrong'
+];
+
+function randElem(arr) {
+  return arr[Math.floor(Math.random() * arr.length)];
+}
+
+export const AddInboxItem = {
+  name: 'add-inbox-item',
+
+  matches: (str) => {
+    return true;
+  },
```

Comment:
Consider using the shorthand method definitions:

    matches(str) {
      return true;
    }